README.md

```diff
@@ -1,3 +1,21 @@
 This repository contains Guix attestations for releases of Bitcoin Core.
 
 See https://github.com/bitcoin/bitcoin/blob/master/doc/release-process.md#building on how to build the release with Guix and create an attestation.
+
+## Overall process
+
+The Guix build consists of two stages:
+
+- In the first stage (`noncodesigned`), people compile the binaries from source.
+- Then, code signatures for Windows and MacOS are generated from the binaries that were produced in the first stage, and distributed to the builders.
```

Comment:
Maybe link to the detached sigs repo https://github.com/bitcoin-core/bitcoin-detached-sigs ?

Reply:
Sure, happy to add that, though this is really only meant as a high-level overview not detailed instructions.

Reply:
Thinking of it, it might make sense to move that line *under* here.

Reply:
I've added a link (made the word "distributed" a link to that repo), and moved the detailed process link to below the overall process description.